platooning_control/src/platoon_control.cpp

```diff
@@ -13,311 +13,427 @@
-        pnh_->param<double>("wheelBase", config.wheelBase, config.wheelBase);
-        pnh_->param<double>("correctionAngle", config.correctionAngle, config.correctionAngle);
-        pnh_->param<double>("integratorMax_pp", config.integratorMax_pp, config.integratorMax_pp);
-        pnh_->param<double>("integratorMin_pp", config.integratorMin_pp, config.integratorMin_pp);
-        pnh_->param<double>("Ki_pp", config.Ki_pp, config.Ki_pp);
-
-        // Global params (from vehicle config)
-        pnh_->getParam("/vehicle_id", config.vehicleID);
-        // pnh_->getParam("/vehicle_wheel_base", config.wheelBase);
-        pnh_->getParam("/control_plugin_shutdown_timeout", config.shutdownTimeout);
-        pnh_->getParam("/control_plugin_ignore_initial_inputs", config.ignoreInitialInputs);
-
-        pcw_.updateConfigParams(config);
-        config_ = config;
-
-	  	// Trajectory Plan Subscriber
-		trajectory_plan_sub = nh_->subscribe<cav_msgs::TrajectoryPlan>("platoon_control/plan_trajectory", 1, &PlatoonControlPlugin::trajectoryPlan_cb, this);
-        
-        // Current Twist Subscriber
-        current_twist_sub_ = nh_->subscribe<geometry_msgs::TwistStamped>("current_velocity", 1, &PlatoonControlPlugin::currentTwist_cb, this);
-
-        // Platoon Info Subscriber
-        platoon_info_sub_ = nh_->subscribe<cav_msgs::PlatooningInfo>("platoon_info", 1, &PlatoonControlPlugin::platoonInfo_cb, this);
-
-		// Control Publisher
-		twist_pub_ = nh_->advertise<geometry_msgs::TwistStamped>("twist_raw", 5, true);
-        ctrl_pub_ = nh_->advertise<autoware_msgs::ControlCommandStamped>("ctrl_raw", 5, true);
-        platoon_info_pub_ = nh_->advertise<cav_msgs::PlatooningInfo>("platooning_info", 1, true);
-
-
-        pose_sub_ = nh_->subscribe("current_pose", 1, &PlatoonControlPlugin::pose_cb, this);
-
-		plugin_discovery_pub_ = nh_->advertise<cav_msgs::Plugin>("plugin_discovery", 1);
-        plugin_discovery_msg_.name = "platoon_control";
-        plugin_discovery_msg_.version_id = "v1.0";
-        plugin_discovery_msg_.available = true;
-        plugin_discovery_msg_.activated = true;
-        plugin_discovery_msg_.type = cav_msgs::Plugin::CONTROL;
-        plugin_discovery_msg_.capability = "control/trajectory_control";
-
-
-        discovery_pub_timer_ = pnh_->createTimer(
-            ros::Duration(ros::Rate(10.0)),
-            [this](const auto&) { plugin_discovery_pub_.publish(plugin_discovery_msg_);
-                                  ROS_DEBUG_STREAM("10hz timer callback called");});
-        
-        ROS_DEBUG_STREAM("discovery timer created");
-        
-        control_pub_timer_ = pnh_->createTimer(
-            ros::Duration(ros::Rate(30.0)),
-            [this](const auto&) { ROS_DEBUG_STREAM("30hz timer callback called"); 
-                                  controlTimerCb();  }); 
-        
-        ROS_DEBUG_STREAM("control timer created ");
+        RCLCPP_DEBUG_STREAM(get_logger(), "returning due to timeout.");
```

Comment:
let's replace all `get_logger()` in this file and others to `rclcpp::get_logger("platoon_control")` to guarantee this logger prints for now. You can also declare member variable for the name and use it throughout.